README.md

```diff
@@ -121,8 +121,6 @@ describe('dao', ()=>{
 [![CircleCI](https://circleci.com/gh/mikhail-angelov/mongo-unit.svg?style=svg)](https://circleci.com/gh/mikhail-angelov/mongo-unit)
 
 I was inspired by [dbUnit](http://dbunit.sourceforge.net) library, which is very popular in java world.
-
```

Comment:
it make sense to mention other alternative,
now it `https://github.com/nodkz/mongodb-memory-server`